sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetReadSupport.scala

```diff
@@ -122,12 +122,15 @@ class ParquetReadSupport(val convertTz: Option[ZoneId],
       keyValueMetaData: JMap[String, String],
       fileSchema: MessageType,
       readContext: ReadContext): RecordMaterializer[InternalRow] = {
+    val caseSensitive = conf.getBoolean(SQLConf.CASE_SENSITIVE.key,
+      SQLConf.CASE_SENSITIVE.defaultValue.get)
```

Comment:
I'm not sure why you need to pass `caseSensitive` across `ParquetRecordMaterializer`, `ParquetRowConverter`. Can't we just get it at `ParquetRowConverter`?

Reply:
Can I get runtime config at `ParquetRowConverter`? I'm not concretely understand it's behavior.

Reply:
`SQLConf.get` works, even in executor sid, see https://github.com/apache/spark/commit/dd37529a8dada6ed8a49b8ce50875268f6a20cba

Reply:
Thanks! I'll update to using SQLConf instead of passing argument across classes.